client/web/src/site-admin/backend.ts

```diff
@@ -316,16 +339,20 @@ export function fetchUserUsageStatistics(args: {
                         id
                         username
                         usageStatistics {
-                            searchQueries
-                            pageViews
-                            codeIntelligenceActions
-                            lastActiveTime
-                            lastActiveCodeHostIntegrationTime
+                            ...UsageStatistics
                         }
                     }
                     totalCount
                 }
             }
+
+            fragment UsageStatistics on UserUsageStatistics {
```

Comment:
To be consistent with other fragment names:

```suggestion
            fragment UserUsageStatisticsFields on UserUsageStatistics {
```

We currently don't have an enforced migration for GraphQL fragment names. If you have ideas about this, let me know.